test/kubetest/k8s.go

```diff
@@ -1386,6 +1412,11 @@ func (k8s *K8s) setIPVersion() {
 	}
 }
 
+// UseIPv6 returns which IP version is going to be used in testing
```

Comment:
Why we need two methods?

Reply:
UseIPv6 flag is needed before NewK8s() call to set EXCLUDED_PREFIXES variable before PrefixServicePod() is called